### PR TITLE
kubeadm install gets netctl from container

### PR DESCRIPTION
**Pull request by chrisplo**
netctl used to be added in the minimal installer, but was removed.

The binary is in the netplugin container that is run on every node, so
instead of including it in the minimal installer, extract it from the
container.

### CODE REVIEW

**Comment by unclejack** [comment]
build PR

File: install/k8s/install.sh [hunk 1]
```diff
@@ -287,6 +295,10 @@ for i in {0..150}; do
 	sleep 2
 	# check contiv netmaster pods
 	$kubectl get pods -n kube-system | grep -v "Running" | grep -q ^contiv-netmaster  && continue
+	break
+done
+for i in {i..150}; do
```
Comment (chrisplo) [inline install/k8s/install.sh]: does not work

**Comment by chrisplo** [comment]
Fixing in #292

**Comment by unclejack** [comment]
#292 was merged. I'll close this now.

**Pull request closed without merging**